mig-agent/agent.go

```diff
@@ -513,7 +513,12 @@ func startRoutines(ctx *Context) (err error) {
 		for msg := range ctx.Channels.NewCommand {
 			err = parseCommands(ctx, msg)
 			if err != nil {
-				log := mig.Log{Desc: fmt.Sprintf("%v", err)}.Err()
+				var log mig.Log
+				if strings.Contains(err.Error(), "signature made by unknown entity") {
+					log = mig.Log{Desc: fmt.Sprintf("%v \n Action rejected -- sent by unknown investigator.", err)}.Err()
```

Comment:
@tydavis is this line feed in here because we are bundling two log lines up in a single message?

I'm wondering if instead of doing this, we should just send the "Action rejected" log message after sending the first one which contains the unknown entity error.

The reason being there are a couple of log output modes (e.g., syslog, windows event log) and I'm not sure what the behavior of these will be with multiline input.